ipaplatform/base/services.py

```diff
@@ -539,4 +539,4 @@ def base_service_class_factory(name, api=None):
 
 # System may support more time&date services. FreeIPA supports chrony only.
 # Other services will be disabled during IPA installation
-timedate_services = ['ntpd', 'chronyd']
+timedate_services = ['ntpd', 'chronyd', 'systemd-timesyncd']
```

Comment:
I don't know if this is safe on Fedora and RHEL.

@abbra ?

Reply:
how could there be both chronyd and systemd-timesyncd running?

Reply:
Right, chronyd is the only time service supported. By adding systemd-timesyncd to the list it will be explicitly disabled and shut down if it is running. So I don't see a problem with it, assuming you're ok with chrony.